conda-forge.yml

```diff
@@ -1,5 +1,3 @@
-bot:
-  automerge: true
```

Comment:
It doesn't seem like the dependencies are stable enough for the automerge bot to be safe.

Reply:
Automerge still requires a label manually added to the PR, right?